### DISCUSSION TITLE
Welcome to nff-go Discussions!

### DISCUSSION THREAD

**Discussion by aregm** [aregm/nff-go#721] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by swatson314159** [comment]
Hello, I use NFF-Go at Comcast.  I know DPDK is strongest for high volumes of small packets, but I found even with my use case handling ~4million packets/second at ~1400 bytes each I could use NFF-Go to handle the load.  I couldn't find any way to get that many packets through the Linux kernel network stack. Granted, I'm using a kernel that doesn't support XDP.
